builder/googlecompute/config_test.go

```diff
@@ -712,6 +712,72 @@ func TestConfigPrepareImageArchitecture(t *testing.T) {
 	}
 }
 
+func TestConfigInstanceMaxRunDurationSeconds(t *testing.T) {
+	cases := []struct {
+		Keys   []string
+		Values []interface{}
+		Err    bool
```

Comment:
Very nitpicky but in terms of self-documentation, it would imho be better if this field was called `ExpectErr` instead of just `Err` to convey the intent that this is the expected outcome of the test here.